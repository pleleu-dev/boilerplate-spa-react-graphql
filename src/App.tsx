import Button from '@material-ui/core/Button';
import React from 'react';
import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Setting up project mix-and-match
        </p>
        <Button variant="contained" color="primary">
          Hello World
        </Button>
      </header>
    </div>
  );
}

export default App;
