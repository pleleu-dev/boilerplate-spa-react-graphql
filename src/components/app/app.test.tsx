import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from './app';



configure({ adapter: new Adapter() });

test('renders the component', () => {
  const component = shallow(<App />);
  expect(component).toMatchSnapshot();
});
