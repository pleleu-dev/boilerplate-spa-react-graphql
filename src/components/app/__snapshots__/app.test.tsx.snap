// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<BrowserRouter>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(Grid))
        item={true}
        xs={12}
      >
        <WithStyles(ForwardRef(Grid))
          container={true}
          justify="center"
          spacing={2}
        >
          <WithStyles(ForwardRef(Grid))
            item={true}
          >
            <WithStyles(ForwardRef(Typography))
              variant="h6"
            >
              <img
                alt="logo"
                className="logo"
                src="logo.png"
              />
            </WithStyles(ForwardRef(Typography))>
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Button))
        color="inherit"
      >
        Login
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={12}
    >
      <WithStyles(ForwardRef(Grid))
        container={true}
        justify="center"
        spacing={2}
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
        >
          <WithStyles(ForwardRef(Typography))
            variant="h6"
          >
            <Link
              to="/"
            >
              Home
            </Link>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
        >
          <WithStyles(ForwardRef(Typography))
            variant="h6"
          >
            <Link
              to="/about"
            >
              About
            </Link>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
        >
          <WithStyles(ForwardRef(Typography))
            variant="h6"
          >
            <Link
              to="/dashboard"
            >
              Dashboard
            </Link>
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Toolbar))>
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Home />
    </Route>
    <Route
      path="/about"
    >
      <About />
    </Route>
    <Route
      path="/dashboard"
    >
      <Dashboard />
    </Route>
  </Switch>
</BrowserRouter>
`;
